lib/iris/_concatenate.py

```diff
@@ -788,8 +765,9 @@ def concatenate(self):
         """Concatenates all the source-cubes registered with the
         :class:`_ProtoCube` over the nominated common dimension.
 
-        Returns:
-            The concatenated :class:`iris.cube.Cube`.
+        Returns
+        -------
+        The concatenated :class:`iris.cube.Cube`.
```

Comment:
Indeed.  However the existing docstrings often do no include the type.  Using a one line like this still renders ok although not aligned 100% to numpydoc style.  This can be improved in the wider code base later.